reconcile/test/test_utils_slack_api.py

```diff
@@ -1,19 +1,25 @@
+import json
 from collections import namedtuple
 from unittest.mock import call, patch
 
+import httpretty
 import pytest
+from slack_sdk.errors import SlackApiError
 
-from reconcile.utils.slack_api import SlackApi, SlackAPIRateLimitedException, \
-    SlackAPICallException
+from reconcile.utils.slack_api import SlackApi, MAX_RETRIES, \
+    UserNotFoundException
 
 
 @pytest.fixture
 def slack_api():
     with patch('reconcile.utils.slack_api.SecretReader', autospec=True) as \
             mock_secret_reader, \
-            patch('reconcile.utils.slack_api.SlackClient', autospec=True) as \
+            patch('reconcile.utils.slack_api.WebClient', autospec=True) as \
```

Comment:
I tend to prefer `patch.object`
```suggestion
            patch.object(reconcile.utils.slack_api, 'WebClient', autospec=True) as \
```
but no biggie.

Reply:
I've thought about this for a while and can't think of any immediate downsides.  I like typing fewer strings.  I'll change this and replace the context manager usage.